tools/validators/instance_validator/main.py

```diff
@@ -12,40 +12,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import instance_parser 
-import sys
+"""Main of the instance validator."""
```

Comment:
but what does it do?  In general, header comments should cover:
1. "what does this do?
2. "how is it done?" (high level description - may not always be necessary)
3. "why is it important?"

basically, a user should be able to walk through the code reading only the header comments and fully understand what the application does and how it's specific functionality relates to the business needs that the application serves.  

Note: since this is a file with just a main method, the header documentation can be at the top of the file if you want (I don't have a strong preference of where it goes)

Reply:
Done